File: c/2-control structures/workout/csw-5.c
#include <stdio.h>
int main() {
    int a;
    scanf("%d\n",&a);
    if(a>=1&&a<=3)
        {
            printf("1\n");
            printf("1\n");
            
        }
    else if(a>=4&&a<=6)
    {
        printf("2\n");
        printf("1\n");
        
    }
    else if(a>=7&&a<=9)
    {
         printf("3\n");
         printf("2\n");
         printf("1\n");
    }
     else if(a==10)
    {
         printf("5\n");
         printf("3\n");
         printf("2\n");
         printf("1\n");
    }
    else{
         printf("Invalid input");
    }
}

File: java/3-control-statements/workout/csw-5.java
import java.util.Scanner;
public class A{
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        if(a<1){
            System.out.println("Invalid input");
        }
        for(int  i=a;i>=1;i--)
        {
            System.out.println(i);
        }
    }
}